### ISSUE TITLE
岭回归

### ISSUE THREAD

**Issue by tonyhuang33** [pavancm/CONTRIQUE#20]
我想问一下你们是怎么实现，分批次图像质量评估数据集来训练岭回归网络，最终得到一个训练模型的，我只能复现一个批次一个训练模型，怎么把多个批次的模型整合成一个训练模型呢？你们有相关代码提供吗？

**Comment by pavancm** [comment]
In my current implementation the ridge regression is trained on the entire dataset in one shot (not in batches). Note that I don't combine multiple IQA datasets for training ridge regressor, only a single IQA dataset is used. I haven't tried batch training for ridge regression but it should be possible to do that in PyTorch.

**Comment by tonyhuang33** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by tonyhuang33** [comment]
thanks!!

**Comment by tonyhuang33** [comment]
Thanks for replying，So you put all the img features into one feature file? how?

**Comment by tonyhuang33** [comment]
thanks for replying , i will try to concatenated these features into one feature file.

**Comment by tonyhuang33** [comment]
i have a question to ask. First, When I use your regression model for IQA testing, how can I determine if the test set I am using contains your training data?

**Comment by tonyhuang33** [comment]
i have a question to ask.  When I use your regression model for IQA testing, how can I determine if the test set I am using contains your training data?

**Comment by pavancm** [comment]
The data we used to train the model is from different public datasets and we have listed the names of those datasets in the paper. During testing please make sure there is no overlap of images from these datasets.

**Comment by tonyhuang33** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by tonyhuang33** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
